### PR TITLE
Update Command_tag.java

### PR DESCRIPTION
**Pull request by ownre**

### CODE REVIEW

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_tag.java [hunk 1]
```diff
@@ -11,7 +11,7 @@
 import org.bukkit.command.CommandSender;
 import org.bukkit.entity.Player;
 
-@CommandPermissions(level = AdminLevel.OP, source = SourceType.BOTH)
+@CommandPermissions(level = AdminLevel.ALL, source = SourceType.BOTH)
```
Comment (DUFCLiam) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_tag.java]: Change back to Op

**Comment by rovertdude** [comment]
I know what he did.
He made it all for a reason; it's good for usage of /tag off when you get smited for having an illegal tag and you are not op.

**Comment by FoxIshDaBest** [comment]
Ill just change the AdminLevel.OP to ALL really quick.

**Comment by DUFCLiam** [comment]
@FoxIshDaBest You mean ALL to OP haha

**Comment by FoxIshDaBest** [comment]
No, I dont...

**Pull request closed without merging**